release/nightly_tests/dataset/stable_diffusion_benchmark.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+import json
+import os
+from timeit import default_timer as timer
+from typing import Dict
+
+import numpy as np
+import torch
+from diffusers import StableDiffusionImg2ImgPipeline
+
+import ray
+
+DATA_URI = "s3://air-example-data-2/300G-image-data-synthetic-raw-parquet/"
+BATCH_SIZE = 20
```

Comment:
is this the largest batch size that can be used?

Reply:
No, but I ran some experiments, and it seems like throughput declines at higher batch sizes. Also, we achieve virtually 100% GPU utilization with lower batch sizes, so I don't think it'll be an issue.